Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/computers_shipyard.yml

```
@@ -166,6 +166,22 @@
       Invalid: 0.30
     ignoreBaseSaleRate: true # Fixed 70% sale rate.
     canTransferDeed: false
+  - type: Destructible
+    thresholds:
+        - trigger:
+            !type:DamageTrigger
+            damage: 250
+          behaviors:
+            - !type:PlaySoundBehavior
+              sound:
+                collection: GlassBreak
+            - !type:SpawnEntitiesBehavior
+              spawn:
+                PirateComputerBroken:
+                  min: 1
+                  max: 1
+            - !type:DoActsBehavior
+              acts: [ "Destruction" ]
```

```suggestion
              acts: [ "Destruction" ]
  - type: Construction
    graph: NFPirateShipyard
    node: repaired
```
I believe you need to have the construction graph set up in the entity before it'll actually work in the construction graph.

Didn't test this, for what it's worth.